.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+
+_Please include a summary of the changes, relevant motivation and context_.
+
+# Issue
+
+* _Redmine Ticket URL_ or _Github Issue ID_.
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code.
+- [ ] My code follows the style guidelines of this project.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] New and existing tests pass locally with my changes.
+- [ ] This branch is up to date with the main branch.
+- [ ] If this change is a core feature, I have added thorough tests.
+- [ ] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories.
```

Comment:
Will you add a section to clarify these dependencies?

Reply:
It will be an internal document, since currently we don’t have a _code of conduct_ or a _contributing guideline_.